cmd/root.go

```diff
@@ -0,0 +1,13 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	RootCmd = &cobra.Command{
+		Use:   "e2e-harness",
+		Short: "Harness for custom kubernetes e2e testing",
+		Long:  ``,
```

Comment:
Is there a particular reason to not use " here?

Reply:
(this is more for my curiosity)

Reply:
Don't remember where did i got this from, i think that being the Long description backticks make sense if it's going to have more than one line, not the case :) using " now

Reply:
Given that it is not being used I've removed the `Long` field